data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+			choice
+				`	(Shout now to surprise him.)`
+					goto start
+				`	(Get into position and try to prepare.)`
+			`	You carefully observe your surroundings, and decide to head toward one of the smaller circles. Everything is simulated to resemble an urban environment, probably because the original Unfettered had to protect their cities from the Korath. You feel the adrenaline of the coming battle surging into your veins.`
+			label start
+			`	You shout and try to shoot him... Except you completely miss him; you weren't expecting so much recoil from such a small pistol. It also seems to be wildly inaccurate. Obviously, your opponent knew this, and he is already rushing your way.`
+			choice
+				`	(Stay here and get ready for your next shot.)`
+					goto ready
+				`	(Go for the cover on the right.)`
+			branch lucky
+				random < 20
+			`	You try to head for cover, but he's too fast, and you receive a shot on your leg, making you fall. It hurts, but nothing seems to be broken, and soon after you cannot feel anything in your leg at all. You opponent comes to you and helps you to get up. "Pain is a good teacher sometimes," he says. Then he goes back to fight with his friends, seemingly in teams.`
+				goto heyi
+			label lucky
+			action
+				outfit "Dual Pulse Pistols" 1
+			`	You actually manage to reach cover, thanks to him stumbling a bit on the approach. He was probably anxious to beat you... Now that you are behind cover, you have a strategic advantage, and he misses his shots. It seems that the pistol quickly runs out of power, and using this opportunity you shoot back at him. Most of your shots miss, but because he cannot shoot back and you are well protected anyway, you can take the time to aim precisely. You manage to shoot his arm and he drops his pistol. He does not shout at all when getting shot, simply sits on the ground on that position, showing he surrenders. You get out of your cover and grab the other pistol. "You can keep them; you've earned it. This was a fair fight, Captain! I'd like to try again some time, and maybe get my pistols back..." He then goes away and sits on the side, probably thinking about what mistakes he made.`
+				goto heyi
+			label ready
+			`	You try to shoot him, but h's too fast and you too inexperienced. You have made yourself an easy target for him by standing still; you fall as you receive a shot to your shoulder. It hurts, but nothing seems to be broken, and soon after you cannot feel anything in your arm at all. You opponent comes to you and helps you to get up. "Pain is a good teacher sometimes," he says. Then he goes back to fight with his friends, seemingly in teams.`
+			label heyi
+			`	It seems like your fight has attracted the attention of many of the Unfettered, including He Yi. He comes to speak to you. "That was a nice fight, especially considering this must be your first!"`
+			choice
+				`	"Can I help you with something?"`
+					goto speak
+				`	"Well, I lost."`
```

Comment:
Seems like this could be set up with branches to make this choice available only for those who actually lost. Would that be difficult?

Reply:
Soon no, with the PR to hide given choices depending on the player's conditions
I really wish that PR would be merged sooner rather than later, in the meantime it wont take too much work to do what you say

Reply:
Eh, I'd say there's literally no reason for someone to lie. What would they gain?